components/StyledComponents.js

```diff
@@ -252,3 +252,94 @@ export const SmallRoundButton = styled.button`
     color: white;
   }
 `;
+
+export const Checkbox = styled.input`
+  position: relative;
+  float: right;
+  margin: 0 0.2rem 0.3rem 0;
+
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
```

Comment:
You do not need to prefix any properties in styled components, any required prefixes will be created automatically.

```suggestion
  appearance: none;
```